### PR TITLE
use native k8s name

### PR DESCRIPTION
**Pull request by xiaomuqiao**
change names,  support to use native k8s name

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+## Version Description
 
-## Setup Instructions
+```
+    golang：1.6+
+    docker：(Suggested)1.10.1
+    kubernetes：1.2+
+```
 
+## Setup Instructions
```
Comment (gaocegege) [inline docs/setup.md]: s/Instructions/Instruction

File: docs/setup.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
 # Setup
 
+## Version Description
```
Comment (superxi911) [inline docs/setup.md]: Dependence

Comment (xiaomuqiao) [inline docs/setup.md]: 已改

**Comment by superxi911** [comment]
LGTM

**Pull request merged**